packages/@atjson/source-commonmark/test/extensions.test.ts

```diff
@@ -79,3 +84,27 @@ describe("strikethrough", () => {
     `);
   });
 });
+
+describe("tables", () => {
+  const tableExample = `
+| name     | age | job     | [*notes*](https://ja.wikipedia.org/wiki/ダンジョン飯)                                                                                    |
+|:-------- | ---:| ------- |:------------------------------------------------------------------------------------------------------------------------------------------------:|
+| laios    | 20  | fighter | ちょっと変な but earnest person. He *really __really__* likes monsters                                                                              |
```

Comment:
🤔 this looks like the maybe the kanji is causing some offset with the text length? I'm not sure if it matters, tbh, but just wanted to note it.